proposals/csharp-7.0/tuples.md

```diff
@@ -290,37 +311,37 @@ System.ValueTuple<int, int> vt = t;  // identity conversion
 
 t2.moo = 1;
 ```
+end example\]
 
-> note:
->
-> An element name at one position on one side of a conversion, and the same name at another position on the other side almost certainly have bug in the code:
+In teh case in which an element name at one position on one side of a conversion, and the same name at a different position on the other side, the copmpiler shall issue a warning.
```

Comment:
nit:
```suggestion
In the case in which an element name at one position on one side of a conversion, and the same name at a different position on the other side, the compiler shall issue a warning.
```

Reply:
Got it; that's probably my most common typing mistake!

Reply:
me too.